MinPlatformPkg/Test/Library/TestPointCheckLib/TraditionalMmTestPointCheckLib.c

```diff
@@ -66,6 +60,30 @@ TestPointDumpUefiMemoryMap (
   IN  BOOLEAN               DumpPrint
   );
 
+EFI_STATUS
+EFIAPI
+TestPointSmmReadyToLockSmmMemoryAttributeTableFunctional (
+  VOID
+  );
+
+EFI_STATUS
+EFIAPI
+TestPointSmmReadyToLockSecureSmmCommunicationBuffer (
+  VOID
+  );
+
+/**
+  The MM library constructor.
+  The function does the necessary initialization work for this library
+  instance.
+  @retval     EFI_SUCCESS       The function always return EFI_SUCCESS.
+**/
+EFI_STATUS
+EFIAPI
+MmTestPointCheckLibConstructor (
+  VOID
+  );
```

Comment:
`MmTestPointCheckLibConstructor()` is another example of a function that should ideally be in a library local header file. I'd recommend dropping `EFIAPI` since the function is now only called internally within the library.

Reply:
Updated.